standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestListPartitions.java

```diff
@@ -266,7 +271,9 @@ private void assertCorrectPartitionValuesResponse(List<List<String>> testValues,
     }
   }
 
+  protected void assertPartitionsHaveCorrectParams(List<Partition> partitions) {
 
```

Comment:
Is this still WIP? I believe you wanted to check if partitions have correct params correct?

Reply:
The check for partition params happens here:
https://github.com/apache/hive/blob/84583f6fa1c779908852c49510c03495534f2e03/ql/src/test/org/apache/hadoop/hive/metastore/TestListPartitionsWithXIncludeParams.java#L124-L133

The `TestListPartitionsWithXIncludeParams` is a child class of the `TestListPartitions`, so it will overwrite this method. When runing `TestListPartitionsWithXIncludeParams`, it will also check the partition params of the tests in `TestListPartitions`.